resources/devices/mobile-phone/htc.json

```diff
@@ -2183,32 +2170,6 @@
             "isTablet": "false"
         }
     },
-    "HTC Wildfire S": {
```

Comment:
These two were consolidated to a single "HTC A510" device (also the Wildfire S).